Devscord.DiscordFramework/Framework/Commands/Services/BotCommandsPropertyConversionService.cs

```diff
@@ -12,15 +16,23 @@ public class BotCommandsPropertyConversionService
 
         public object ConvertType(string value, BotCommandPropertyType type)
         {
-            return type switch
+            try
```

Comment:
wcześniej masz etap walidacji
ten kod jest odpalany dopiero jak walidacja przejdzie prawidłowo

więc jeśli coś sie wywala tutaj, to trzeba naprawić to tam
a nie bawić sie we wrzucanie try catch